### DISCUSSION TITLE
ability to import setup.py projects

### DISCUSSION THREAD

**Discussion by deronnax** [python-poetry/poetry#8437] [Ideas]
That would be truly awesome if poetry was able to import and generate a poetry-pyproject.toml file from setup.py projects.
PDM has this through `pdm import`. There are tools to migrate/convert setup.py/setup.cfg to PEP 621 pyproject.toml but there are none for poetry pyprojects.toml.

**Comment by radoering** [comment]
That's a typical use case for a plugin. (If no one has written such a plugin yet, it probably wasn't important enough.)
